dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/scheduling/JobScheduler.java

```diff
@@ -76,7 +76,7 @@ public class JobScheduler implements Runnable, JobRunner {
    *   <li>Leaves as much time as possible otherwise to get through the work of triggering jobs
    * </ul>
    */
-  private static final int LOOP_SECONDS = 20;
+  private static final int LOOP_SECONDS = 20000000;
```

Comment:
???

Reply:
@jbee Ha, that's temporary! When I'm debugging I sometimes find it very helpful to watch my Postgres log in real time to see what's happening in the database. (I log all statements all the time and rotate the log by day of week so it doesn't fill up my disk.) When I don't need the job scheduler (which is nearly all the time), I change this value so it's not dumping in a bunch of queries three times a minute. It make the log *much* easier to follow!